druid-shell/src/windows/application.rs

```diff
@@ -14,10 +14,121 @@
 
 //! Windows implementation of features at the application scope.
 
+use std::ptr;
+
+use winapi::shared::minwindef::FALSE;
+use winapi::shared::minwindef::UINT;
+use winapi::shared::ntdef::{LPWSTR, WCHAR};
+use winapi::shared::winerror::ERROR_SUCCESS;
+use winapi::um::errhandlingapi::*;
```

Comment:
*Generally* I prefer listing the items actually imported rather than using wildcards, though it can get annoying.